node/resolvers/Mutations/Organizations.test.ts

```diff
@@ -0,0 +1,144 @@
+import * as casual from 'casual'
+import {
+  anything,
+  capture,
+  instance,
+  mock,
+  reset,
+  spy,
+  verify,
+  when,
+} from 'ts-mockito'
```

Comment:
We usually just use jest directly at VTEX for [mocking](https://jestjs.io/docs/mock-functions), it's the most popular tool out there, and the most complete as well. Plus, we're already using jest to run the tests, so it's one less dependency.

Let me know if you had trouble while using jest for mocking though, and we can either discuss here or setup some time for a pair programming where I can show these test tools for node.js/js